src/lib/slide-toggle/slide-toggle.scss

```diff
@@ -107,7 +107,7 @@ $mat-slide-toggle-bar-track-width: $mat-slide-toggle-bar-width - $mat-slide-togg
 
   // Once the thumb container is being dragged around, we remove the transition duration to
   // make the drag feeling fast and not delayed.
-  &.mat-dragging, &:active {
+  &.mat-dragging, &:active, ._mat-animation-noopable & {
```

Comment:
Why does `-noopable &` need this `cursor-grabbing`?

Reply:
ops, good catch. It's fixed now.